### ISSUE TITLE
When the Description content of the PythonFunctionTools component contains curly braces' {} ', an error will be reported

### ISSUE THREAD

**Issue by zhengxiang5965** [logspace-ai/langflow#774]
**Describe the bug**
When the Description content of the PythonFunctionTools component contains curly braces' {} ', an error will be reported
**Browser and Version**
 - Browser edge
 - Version 0.4.8
**To Reproduce**

**Screenshots**
![Dingtalk_20230816184613](https://github.com/logspace-ai/langflow/assets/31379313/35c665e4-d8fc-43a5-ae78-693b3bf2fb26)

**Additional context**
Description content is 
[{"name": "", "": {"age": ""}}]

**Comment by zhengxiang5965** [comment]
@ogabrielluiz @lucaseduoli @anovazzi1

**Comment by ogabrielluiz** [comment]
Hey @zhengxiang5965 

This is a problem because the Tool's description is used in the Agent's Prompt. This isn't a bug as this is how it works in LangChain. 

You could describe it in plain English like `Use this tool passing a list of python dictionaries containing ...`.

**Comment by zhengxiang5965** [comment]
> Hey @zhengxiang5965
> 
> This is a problem because the Tool's description is used in the Agent's Prompt. This isn't a bug as this is how it works in LangChain.
> 
> You could describe it in plain English like .`Use this tool passing a list of python dictionaries containing ...`
@ogabrielluiz  

The officially recommended prompt for the llm qwen-chat-7b I am using includes descriptions formatted with JSON. qwen-chat-7b itself supports tool descriptions that include formatting JSON. Is there a way to fix this issue by escaping the content and sending it to llm
